### PR TITLE
Change i915.domain_plane_owners for Hybrid scenario

### PR DESCRIPTION
**Pull request by fuzhongl**

### CODE REVIEW

**Comment by binbinwu1** [comment]
@yakuizhao  Could you help to review the patch?

**Comment by binbinwu1** [comment]
@fuzhongl  Could you rephrase the commit message? it is not clear for the purpose of the change.

**Comment by yakuizhao** [comment]
It seems that the pipe A is already assigned to UOS. But the sos is still using 4 planes.
This is incorrect.

**Comment by fuzhongl** [comment]
> It seems that the pipe A is already assigned to UOS. But the sos is still using 4 planes.
> This is incorrect.
@yakuizhao  with following setting, PipeA is assigned to SOS; not UOS.
Please refer https://projectacrn.github.io/latest/user-guides/kernel-parameters.html
i915.avail_planes_per_pipe and i915.domain_plane_owners section

"i915.avail_planes_per_pipe=0x01010F "	\
"i915.domain_plane_owners=0x011111110000 " \

**Comment by fuzhongl** [comment]
> @fuzhongl Could you rephrase the commit message? it is not clear for the purpose of the change.

Done! Please help to review again. 
thanks!

**Comment by acrnsi** [comment]
OK to verify

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by fuzhongl** [comment]
> @fuzhongl please also update the commit message of the acrn-config patch.
> And then I will approve after Yakuai @yakuizhao Review+1

@binbinwu1 Updated commit message of the acrn-config patch.
Thanks!

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>
